net/sshuttle/Portfile

```diff
@@ -4,25 +4,25 @@ PortSystem          1.0
 PortGroup           github 1.0
 PortGroup           python 1.0
 
-github.setup        sshuttle sshuttle 1.0.1 v
-fetch.type          git
+github.setup        sshuttle sshuttle 1.0.2 v
 revision            0
 
-maintainers         {gmail.com:herby.gillot @herbygillot} openmaintainer
-categories          net
 description         Transparent proxy server that works as a poor man's VPN.
+
 long_description    Transparent proxy server that works as a poor man's VPN. \
                     Forwards over ssh. Doesn't require admin. Works with \
                     Linux and MacOS. Supports DNS tunneling.
 
+maintainers         {gmail.com:herby.gillot @herbygillot} \
+                    openmaintainer
+
+categories          net
 platforms           darwin
 license             LGPL-2
 homepage            https://sshuttle.readthedocs.io/en/stable/
+fetch.type          git
```

Comment:
also, why can you not just get the .tar.gz file?

Reply:
The build process expects to a git checkout.  That's why it uses `setuptools_scm`: while setting up the Python module, that library provides some information through `git`.

Reply:
okay, we might have had this discussion before for this or another port ;)